ansible/checker.yml

```diff
@@ -0,0 +1,48 @@
+# This playbook deploys a simple stand alone checker for SAP/HANS.
+
+- hosts: all
+
+  tasks:
+    - name: include load_facts
+      import_role:
+        name: load_facts
+      tags:
+        - always
+
+    - name: include 1.1.1 remediate
```

Comment:
Let's name it as `Include 1.1.1`.
We are going to avoid the usage of `remediate` word by now